proposal/comp110_architecture_proposal.tex

```diff
@@ -6,72 +6,131 @@
+		\item[Comments:] This is an informative article on the more ticker parts if the level design, I believe this to be relevant to the point as just finding techniques to finish the level does not contribute to the players enjoyment of the game and finding harder routes is a way of making the player enjoy it more. cited by over 40 people although it is not many it still shows that a few dozen people have. This was found through the internet and key word searches.
+	\end{description}
+	
+	\section*{Paper 2}
+	\begin{description}
+		\item[Title:] Empirical evaluation of procedural level
+		generators for 2D platform games
+		\item[Citation:] \cite{empirical}
+		\item[Abstract:] ''Context. Procedural content generation (PCG) refers to algorithmical creation of game content (e.g.
+		levels, maps, characters). Since PCG generators are able to produce huge amounts of game content, it
+		becomes impractical for humans to evaluate them manually. Thus it is desirable to automate the
+		process of evaluation.
+		Objectives. This work presents an automatic method for evaluation of procedural level generators for
+		2D platform games. The method was used for comparative evaluation of four procedural level
+		generators developed within the research community.
+		Methods. The evaluation method relies on simulation of the human player's behaviour in a 2D
+		platform game environment. It is made up of three components: (1) a 2D platform game Infinite Mario
+		Bros with levels generated by the compared generators, (2) a human-like bot and (3) quantitative
+		models of player experience. The bot plays the levels and collects the data which are input to the
+		models. The generators are evaluated based on the values output by the models. A method based on
+		the simple moving average (SMA) is suggested for testing if the number of performed simulations is
+		sufficient.
+		Results. The bot played all 6000 evaluated levels in less than ten minutes. The method based on the
+		SMA showed that the number of simulations was sufficiently large.
+		Conclusions. It has been shown that the automatic method is much more efficient than the traditional
+		evaluation made by humans while being consistent with human assessments.''
+		\item[Web link:] \url {http://www.diva-portal.org/smash/get/diva2:831319/FULLTEXT01.pdf}
+		\item[Full text link:] \url {http://www.diva-portal.org/smash/get/diva2:831319/FULLTEXT01.pdf}
+		\item[Comments:] Cited by over 300 people this is a more credible source. I believe this is relevant because it introduces the PCG and how it can create thousands of levels and then have a bot run through these levels and collects data and evaluates the results. This was found through the internet and key word searches.
+	\end{description}
+	
+	\section*{Paper 3}
+	\begin{description}
+		\item[Title:] Dynamically adjusting game-play in 2D Platformers using
```

Comment:
This is an unpublished undergraduate project. It is not an appropriate source. Look through the reference list to see if any academic literature that you can use has been referenced.